mmdet/utils/misc.py

```diff
@@ -2,6 +2,12 @@
 from mmcv import DictAction
 
 
+DOMAIN_CUSTOM_OPS_NAME = 'org.openvinotoolkit'
+
+def add_domain(name_operator: str) -> str:
+    return DOMAIN_CUSTOM_OPS_NAME + '::' + name_operator
```

Comment:
IMO, `mmdet/utils/deployment/` is a better place for this code.

Reply:
Done.